hivemind/p2p/servicer.py

```diff
@@ -41,35 +42,65 @@ class ServicerBase:
       to calls to the remote peer.
     """
 
```

Comment:
The changes to `Servicer` are:

1. Support classes `ServicerB` inherited like `ServicerBase -> ServicerA -> ServicerB`
2. Make `get_stub()` a classmethod, so you don't need to a servicer yourself to call a method of another peer (used in tests)
3. Improve reflection:
  - Allow any parameter name for `TInputProtobuf` in `rpc_*` methods (previously, only `request` was allowed)
  - Use more robust type annotation parsing (`typing.get_type_hints()` instead of manually parsing string annotations in some cases)